src/content/self-hosted/helm-configuration.mdx

```diff
@@ -703,7 +703,7 @@ quickstarts:
 Enables [resource quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) at the namespace level.
 
 - `resources`: Limits the number of resources that a user can create.
-  - `maxNamespaces`: Maximum number of namespaces.
+  - `maxNamespaces`: Maximum number of namespaces that a user can create.
```

Comment:
I would document that this quota doesn't apply to `okteto-bot`. We can also mention it in the Admin tokens section

Reply:
Should we wait until that work is complete or should we have it ready to ship for 1.19?

Reply:
Oh yes, I was confused. Yes, we can merge this fix and wait for the other feature to be implemented

Reply:
Looks like @ifbyol merged the PR. Making the update!